multimedia/templates/multimedia/book_list.html

```diff
@@ -0,0 +1,19 @@
+{% extends "multimedia/book_base.html" %}
+{% load staticfiles %}
+
+{% block content %}
+{{ block.super }}
```

Comment:
I put this in most of my templates if I am building a nested template. Or else it will overwrite the its parent content. 

Not about common or good practice here. But rather it is about what I wanted to achieve in these templates.